torch/quantization/observer.py

```diff
@@ -880,9 +894,13 @@ def forward(self, x_orig):
         if min_val.numel() == 0 or max_val.numel() == 0:
             min_val = torch.min(x)
             max_val = torch.max(x)
-            self.min_val = min_val
-            self.max_val = max_val
-            self.histogram = torch.histc(x, self.bins, min=min_val, max=max_val)
+            self.min_val.resize_(min_val.shape)
+            self.min_val.copy_(min_val)
+            self.max_val.resize_(max_val.shape)
+            self.max_val.copy_(max_val)
+            new_histogram = torch.histc(x, self.bins, min=min_val, max=max_val)
```

Comment:
Can use the version with an out argument instead:
torch.histc(input, bins=100, min=0, max=0, out=None)